lib/tbot/workloadidentity/workloadattest/docker.go

```diff
@@ -0,0 +1,92 @@
+/*
+ * Teleport
+ * Copyright (C) 2025  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package workloadattest
+
+import (
+	"net/url"
+	"path/filepath"
+
+	"github.com/gravitational/trace"
+)
+
+// DefaultDockerAddr is the default address we will use to connect to the Docker
+// daemon.
+const DefaultDockerAddr = "unix:///var/run/docker.sock"
+
+// DockerAttestorConfig holds the configuration for the Docker workload attestor.
+type DockerAttestorConfig struct {
+	// Enabled determines whether Docker workload attestation will be performed.
+	Enabled bool `yaml:"enabled"`
+
+	// Addr is the address at which the [Docker Engine API] can be reached.
+	//
+	// It must be in the form `unix://path/to/socket`, as connecting via TCP is
+	// not yet supported. It's also currently assumed that the Docker daemon is
+	// running on the same host as `tbot`.
```

Comment:
While we _could_ support connecting to the Docker daemon over TCP, I've opted not to for now because doing so securely would make the configuration more complex (as you'd need to supply mTLS certs, etc) and I think it's a reasonable simplifying assumption that `tbot` is running on the same host and can access a socket file.

Reply:
Sounds sensible to me for now.